### PR TITLE
Added both fbi cap and fbi coat.

### PR DESCRIPTION
**Pull request by Creatorbot01**
## About the PR
Added fbi coat and fbi cap as fbi requested

## Media
fbi1
fbi2
fbi3

## Requirements

- [x] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

ADMIN:
- add: FBI Cap and FBI Coat. Currently unobtainable.

### CODE REVIEW

**Comment by Creatorbot01** [comment]
that weird why is rsi fail

**Comment by Creatorbot01** [comment]
manage to find out why it fail. forgot a file while commiting. whoops.

**Comment by TheDoctor1977** [comment]
discussion because this should almost definitely use the preexisting sprite instead of this unless fed prefers it

File: Resources/Prototypes/Entities/Clothing/Head/hats.yml [hunk 1]
```diff
@@ -1134,4 +1134,16 @@
...
-    - WhitelistChameleon
\ No newline at end of file
+    - WhitelistChameleon
+
+#fbi addition
+- type: entity
+  parent: [ClothingHeadBase]
+  id: ClothingHeadHatFBI
+  name: FBI cap
+  description: A cap that look cool to wear, remind you of certain someone.
```
Comment (bruhmogus) [inline Resources/Prototypes/Entities/Clothing/Head/hats.yml:1144]: this should be changed to "A cap that looks cool, reminds you of a certain someone."

File: Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml [hunk 2]
```diff
@@ -501,3 +501,26 @@
   parent: [ClothingOuterStorageFoldableBaseOpened, ClothingOuterCoatoldRobo]
   id: ClothingOuterCoatoldRoboOpened
   name: old roboticist lab coat
+
+#fbi addition
+- type: entity
+  parent: [ClothingOuterStorageBase, AllowSuitStorageClothing]
+  id: ClothingOuterFBIcoat
+  name: FBI coat
+  description: There a small label inside the coat which read "mary".
```
Comment (bruhmogus) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml:510]: it's probably best not to tie the item to mary, but if we do the sentence should be "There's a small label inside the coat that reads "Mary."

File: Resources/Textures/Clothing/OuterClothing/Coats/fbisuit.rsi/equipped-OUTERCLOTHING.png
Comment (bruhmogus) [inline Resources/Textures/Clothing/OuterClothing/Coats/fbisuit.rsi/equipped-OUTERCLOTHING.png:1]: it should probably actually say fbi on the back, please remember to use capitals (FBI as opposed to fbi)

Comment (bruhmogus) [inline Resources/Textures/Clothing/OuterClothing/Coats/fbisuit.rsi/equipped-OUTERCLOTHING.png:1]: this goes for all sprites but please brighten the coat and hat ever so slightly so they actually look blue

**Review by bruhmogus — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml (hunk 2 above)
Comment (TheDoctor1977) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml:510]: it should just be untied to mary for general use
```suggestion
  description: A coat used by an old Earth enforcement agency. How did this get all the way up here?
```

File: Resources/Prototypes/Entities/Clothing/Head/hats.yml (hunk 1 above)
Comment (TheDoctor1977) [reply Resources/Prototypes/Entities/Clothing/Head/hats.yml:1144]:
```suggestion
  description: A cap that looks cool, reminds you of a certain someone.
```

**Pull request merged**